apps/web-giddh/src/app/services/general.service.ts

```diff
@@ -413,19 +413,24 @@ export class GeneralService {
     /**
      * This will calculate the position of element
      *
-     * @param {*} element
+     * @param {*} elementTarget
+     * @param {*} element 
      * @returns
      * @memberof DatepickerWrapperComponent
      */
-    public getPosition(element): any {
+    public getPosition(elementTarget, element?: any): any {
         var xPosition = 0;
         var yPosition = 40;
 
-        while (element) {
-            xPosition += (element.offsetLeft - element.scrollLeft + element.clientLeft);
-            yPosition += (element.offsetTop - element.scrollTop + element.clientTop);
-            element = element.offsetParent;
+        while (elementTarget) {
+            xPosition += (elementTarget.offsetLeft - elementTarget.scrollLeft + elementTarget.clientLeft);
+            yPosition += (elementTarget.offsetTop - elementTarget.scrollTop + elementTarget.clientTop);
```

Comment:
add condition here that yPosition only updates if element is not there

Reply:
no need to add as I have passed optional params